protected/controllers/AdminSampleController.php

```diff
@@ -315,6 +315,43 @@ protected function performAjaxValidation($model)
 		}
 	}
 
+    public function actionCheckAttribute()
+    {
+        if (!Yii::app()->request->isAjaxRequest) {
+            throw new CHttpException(400,'Invalid request. An Error occurred');
+        }
+
+        $errorMessage = [];
+        $attributesList = $_POST['attr'];
+
+        foreach (explode('",', $attributesList) as $attributes) {
+            $attributes = str_replace('"', '', $attributes);
+            $attributeData = explode('=', $attributes);
+            if (count($attributeData) === 2) {
+                $attributeData[0] = trim($attributeData[0]);
+                if ($attributeData[0] !== 'longitude' && $attributeData[0] !== 'latitude') {
+                    continue;
+                }
+
+                if(preg_match('/^[^0-9]*$/', $attributeData[1])) {
+                   $errorMessage[] = sprintf('Attribute value for %s doesn\'t match WGS84 decimal format.
+                    For geographic location (country, sea, region) use another attribute name', $attributeData[0]);
+                }
```

Comment:
**question:** Can specify the actual attribute name for the `geographic location (country, sea, region)` in the warning message? And usually the `geographic location (country, sea, region)` is something like, `Guangzhou, China, `London, UK`, the regex seems not really applicable to this case?

Reply:
the goal here is to check if they intend to use the attribute 391 and 392. If there are no numbers in the value, we will inform them that it seems like they may have chosen the wrong attribute. If there are numbers, we will check if it matches the WGS84 format. If not, we will indicate that there will likely be an error. It's been asked not to block the curator and just display a warning message.